src/pages/_app.tsx

```diff
@@ -1,8 +1,10 @@
 import '../styles/globals.css'
 import type { AppProps } from 'next/app'
+import '../components/Header/Header.css'
```

Comment:
O CSS do seu componente Header não faria mais sentido sendo carregando dentro do próprio componente?

Reply:
Sim, mas quando eu tentei isso, acho que fiz algo de errado... Dá esse erro:

./src/components/Header/Header.css
Global CSS cannot be imported from files other than your Custom <App>. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://nextjs.org/docs/messages/css-global

Reply:
é justamente o problema de usar o css da forma que está usando, pois quadno você está usando dessa forma aí, ele carrega todo o seu CSS pra quando um dia, se for renderizada tais componentes com esses ids, aplicar o estilo. 

Quando você usa a abordagem de classe e importa o estilo dentro do componente, o CSS só carrega quando o componente é carregado. Vê esses exemplo aqui:

https://www.w3schools.com/react/react_css.asp

Esse tipo de abordagem pro CSS resolve o seu problema de injeção de estilos :3